Client/electrotech/src/pages/home/Home.js

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import LayoutApp from "../../components/Layout";
+import Grid from "@material-ui/core/Grid";
+import Product from "../../components/Product";
+import { useDispatch } from "react-redux";
+import { Button } from "@material-ui/core";
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const [productData, setProductData] = useState([]);
+  const [categories, setCategories] = useState([
+    "laptops",
+    "smart phones",
+    "home applicants",
+  ]);
+  const [selectedCategory, setSelectedCategory] = useState("laptops");
```

Comment:
u should use Category.laptops as enum or const instead of writing the string here